src/Symfony/Component/Uid/UuidV4.php

```diff
@@ -25,14 +25,38 @@ class UuidV4 extends Uuid
     public function __construct(string $uuid = null)
     {
         if (null === $uuid) {
+            $this->uid = self::generate();
+        } else {
+            parent::__construct($uuid);
+        }
+    }
+
+    public function getTime(): float
```

Comment:
I think it's useful to add those methods here as well because of the new factory.

Reply:
I don't understand what this would mean on a V4 UUID, which is only about randomness.

Reply:
Removed.